client/components/dashboard/dashboard-version-service.js

```diff
@@ -113,14 +113,21 @@ DashboardVersionService.prototype.verifyAndUpdateModel = function(dashboard) {
   // current.
   if (version == this.currentVersion) {
     if (!goog.isDef(dashboard.version)) {
+      goog.asserts.assert(!goog.isNull(version.version));
       dashboard.version = version.version;
     }
   } else {
     let dashboard_version_index = this.versions.indexOf(version);
+    
+    goog.asserts.assert(!goog.isNull(this.currentVersion));
```

Comment:
See previous comment - if this is intended to be a check for "non-null or undefined", making it nullable would avoid the need for these asserts.